src/components/Modal/TodayOotd/TodayOotdModal.tsx

```
@@ -0,0 +1,186 @@
+
+      const sortedItems = [...items].sort((a, b) => {
+        if (a.image === ootdData.image) return -1;
+        if (b.image === ootdData.image) return 1;
+        return 0;
+      });
+      setDisplayList(sortedItems);
+      setIsAddMenuOpen(false);
+    } else if (visible && !ootdData) {
+      setDisplayList([]);
+    }
+  }, [visible, date]);
```

ESLint

[error] React Hook useEffect has a missing dependency: 'ootdData'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)